docs/validators/validator-faq.md

```diff
@@ -10,31 +10,31 @@ This is work in progress. Mechanisms and values are susceptible to change.
 
 ## General Concepts
 
-### What is a validator?
+### What is a Cosmos validator?
 
-The [Cosmos Hub](../gaia-tutorials/what-is-gaia.md) is based on [Tendermint](https://tendermint.com/docs/introduction/what-is-tendermint.html), which relies on a set of validators to secure the network. The role of validators is to run a full-node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators commit new blocks in the blockchain and receive revenue in exchange for their work. They must also participate in governance by voting on proposals. Validators are weighted according to their total stake.
+The [Cosmos Hub](../gaia-tutorials/what-is-gaia.md) is based on [Tendermint](https://tendermint.com/docs/introduction/what-is-tendermint.html) that relies on a set of validators to secure the network. The role of validators is to run a full node and participate in consensus by broadcasting votes that contain cryptographic signatures signed by the validator's private key. Validators commit new blocks in the blockchain and receive revenue in exchange for their work. Validators must also participate in governance by voting on proposals. Validators are weighted according to their total stake.
 
-### What is 'staking'?
+### What is staking?
 
-The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (Atoms) bonded as collateral. These Atoms can be self-delegated directly by the validator or delegated to them by other Atom holders.
+The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (ATOM) bonded as collateral. These ATOM can be self-delegated directly by the validator or delegated to the validator by other ATOM holders.
```

Comment:
These ATOM &#8594; These ATOMs

Reply:
Hi @yaruwangway that's what I used to think too! 
see the approved [Cosmos Word List](https://docs.google.com/spreadsheets/d/15raX8lfzzeeifGrDiXSFTHaEDbsMVlFJzKgnNelihjs/edit#gid=0) that we work from for our docs under the tendermint.com umbrella, including the

Reply:
wow this list is amazing @barriebyron !!!